<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1761</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="one">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>120</y>
     <width>250</width>
     <height>700</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="two">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>120</y>
     <width>250</width>
     <height>700</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="three">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>120</y>
     <width>250</width>
     <height>700</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="four">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>120</y>
     <width>250</width>
     <height>700</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="five">
   <property name="geometry">
    <rect>
     <x>1500</x>
     <y>120</y>
     <width>250</width>
     <height>700</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="weather">
        <property name="text">
         <string>마스크</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignJustify|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="now_temperature">
          <property name="font">
           <font>
            <family>휴먼둥근헤드라인</family>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>현재 기온</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="now_temperature_maxmin">
          <property name="font">
           <font>
            <family>휴먼둥근헤드라인</family>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>최저 최고</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="mask">
        <property name="text">
         <string>마스크</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignJustify|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="dust">
          <property name="font">
           <font>
            <family>휴먼둥근헤드라인</family>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>미세먼지</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="dust_ui">
          <property name="font">
           <font>
            <family>휴먼둥근헤드라인</family>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>초미세먼지</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>481</x>
     <y>30</y>
     <width>201</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <widget class="QLabel" name="dateEdit">
      <property name="font">
       <font>
        <family>휴먼둥근헤드라인</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="point_1">
      <property name="font">
       <font>
        <family>휴먼둥근헤드라인</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>30</y>
     <width>201</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_10">
    <item>
     <widget class="QLabel" name="dateEdit_2">
      <property name="font">
       <font>
        <family>휴먼둥근헤드라인</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="point_2">
      <property name="font">
       <font>
        <family>휴먼둥근헤드라인</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>30</y>
     <width>201</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_11">
    <item>
     <widget class="QLabel" name="dateEdit_3">
      <property name="font">
       <font>
        <family>휴먼둥근헤드라인</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="point_3">
      <property name="font">
       <font>
        <family>휴먼둥근헤드라인</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>1260</x>
     <y>30</y>
     <width>201</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_12">
    <item>
     <widget class="QLabel" name="dateEdit_4">
      <property name="font">
       <font>
        <family>휴먼둥근헤드라인</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="point_4">
      <property name="font">
       <font>
        <family>휴먼둥근헤드라인</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_8">
   <property name="geometry">
    <rect>
     <x>1520</x>
     <y>30</y>
     <width>201</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_13">
    <item>
     <widget class="QLabel" name="dateEdit_5">
      <property name="font">
       <font>
        <family>휴먼둥근헤드라인</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="point_5">
      <property name="font">
       <font>
        <family>휴먼둥근헤드라인</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="monitortoken">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>850</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>휴먼둥근헤드라인</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>모니터토큰</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="predict">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>341</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>휴먼둥근헤드라인</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>예측</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_9">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>91</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_14">
    <item>
     <widget class="QLabel" name="dateEdit_6">
      <property name="font">
       <font>
        <family>휴먼둥근헤드라인</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>날짜</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="point_6">
      <property name="font">
       <font>
        <family>휴먼둥근헤드라인</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>평가</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>one</zorder>
  <zorder>two</zorder>
  <zorder>three</zorder>
  <zorder>four</zorder>
  <zorder>five</zorder>
  <zorder>verticalLayoutWidget_3</zorder>
  <zorder>verticalLayoutWidget_4</zorder>
  <zorder>verticalLayoutWidget_5</zorder>
  <zorder>verticalLayoutWidget_6</zorder>
  <zorder>verticalLayoutWidget_7</zorder>
  <zorder>verticalLayoutWidget_8</zorder>
  <zorder>monitortoken</zorder>
  <zorder>predict</zorder>
  <zorder>verticalLayoutWidget_9</zorder>
  <zorder>point_1</zorder>
  <zorder>dateEdit_3</zorder>
  <zorder>point_4</zorder>
  <zorder>point_3</zorder>
  <zorder>dateEdit_4</zorder>
  <zorder>point_5</zorder>
  <zorder>dateEdit_5</zorder>
  <zorder>dateEdit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
